.prettierignore

```diff
@@ -1,2 +1,3 @@
 # Ignore generated files
-packages/jaeger-ui/index.d.ts
\ No newline at end of file
+packages/jaeger-ui/index.d.ts
+scripts
```

Comment:
I think the issue was not with scripts, but with the CHANGELOG that prettier was trying to change (switching between `*` and `-` for bullets)

Reply:
It's fine to ignore the downloaded scripts, but then I'd narrow it to
```suggestion
scripts/release-notes.py
scripts/draft-release.py
```

Reply:
Thanks 😄